src/intl/messages.js

```diff
@@ -134,10 +134,11 @@ export const messages: { [messageId: string]: MessageType } = {
     message: 'Thin Provision',
     description: 'Select list option for qcow / thin provisioned format disk images',
   },
-  diskEditorResizeLabel: 'Extend Size By (GiB)',
+  diskEditorResizeLabel: 'Increase Size By (GiB)',
   diskEditorSizeCantChangeHelp: 'Disk size cannot be extended for this type of disk.',
   diskEditorSizeCreateHelp: 'After you create a disk, you may only extend its size if you make edits.',
-  diskEditorSizeLabel: 'Size (GiB)',
+  diskEditorSizeLabel: 'Current Size (GiB)',
```

Comment:
Changing this key will also change the label on the field for the Create Disk dialog.  To have different text, you'll need another key and adjust the use in `DiskImageEditor`.

For **Create New Disk**, the label should be "Size (GiB)" as it is now.

For **Edit Disk**, the label can say "Current Size (GiB)" or stick with "Size (GiB)".  I could go either way with the one.

@lwrigh , @sgratch - WDYT about the new size / current size field labels?